tests/console/perl_bootloader.pm

```diff
@@ -39,11 +41,20 @@ sub run {
             validate_script_output 'cat /etc/sysconfig/bootloader', qr/LOADER_TYPE="grub2"/;
         }
     }
-    assert_script_run 'pbl --install';
-    assert_script_run 'pbl --config';
-    power_action('reboot', textmode => 1);
-    $self->wait_boot;
-    select_serial_terminal;
+
+    if (is_transactional) {
+        trup_call 'run pbl --install';
```

Comment:
I think you can use `-c #no` in order to chain tr-up commands and avoid reboots

Reply:
Hacks :smile: would be nice to have `trup_call` support chain, can you please open issue ?

Reply:
But I bet we should have more VRs than 2 especially TW